python/model/alarm_clock/alarm_clock.yaml

```diff
@@ -22,6 +22,8 @@ model:
           type: Timestamp
       acceptance:
         - scenario: setTimer
+          given:
```

Comment:
I thought additional changes were going to be made to the model? Am I missing a commit to check or have the changes not made it in yet?